source/whats-new.txt

```diff
@@ -33,7 +33,16 @@ Learn what's new in:
 What's New in 1.11
 --------------------
 
-.. important:: Upgrade to Version 1.11.2 or Higher 
+.. important:: Upgrade to Version 1.11.3 or Higher
+
+   Support for versions 1.11.0 through 1.11.2 has been retracted because
```

Comment:
I: In Jira, Rachelle says we've retracted the versions themselves, not just support for the versions

S:

```suggestion
   Versions 1.11.0 through 1.11.2 of the driver have been retracted because
```